devpack/src/main/java/io/neow3j/devpack/compiler/Compiler.java

```diff
@@ -0,0 +1,234 @@
+                    break;
+                case LDC:
+                    handleLoadConstant(insn, writer);
+                    break;
+                case ICONST_1:
+                    writer.writeByte(OpCode.PUSH1.getValue());
+                    break;
+                case IRETURN:
+                    writer.writeByte(OpCode.RET.getValue());
+                    break;
+                case ASTORE:
+                case ASTORE_0:
+                case ASTORE_1:
+                case ASTORE_2:
+                case ASTORE_3:
+                    handleStoreVariable(insn, writer);
+                    break;
+                case ALOAD:
+                case ALOAD_1:
+                case ALOAD_2:
+                case ALOAD_3:
+                    handleLoadVariable(insn, writer);
+                    break;
+                default:
+                    break;
+            }
+        }
+        System.out.println(Numeric.toHexStringNoPrefix(ms.toByteArray()));
+        System.out.println(ScriptReader.convertToOpCodeString(
+                Numeric.toHexStringNoPrefix(ms.toByteArray())));
+    }
+
+    private static void handleLoadVariable(AbstractInsnNode insn, BinaryWriter writer)
+            throws IOException {
+        assert insn.getType() == AbstractInsnNode.VAR_INSN : "Instruction type doesn't match "
+                + "opcode.";
+        VarInsnNode varInsn = (VarInsnNode) insn;
+        assert varInsn.var <= 255 : "Local variable index to high.";
+        if (varInsn.var <= 6) {
+            writer.writeByte((byte) (OpCode.LDLOC0.getValue() + (byte) varInsn.var));
+        } else {
+            writer.writeByte(OpCode.LDLOC.getValue());
+            writer.writeByte((byte) varInsn.var);
+        }
+    }
+
+    private static void handleStoreVariable(AbstractInsnNode insn, BinaryWriter writer)
+            throws IOException {
+        assert insn.getType() == AbstractInsnNode.VAR_INSN : "Instruction type doesn't match "
+                + "opcode.";
+        VarInsnNode varInsn = (VarInsnNode) insn;
+        assert varInsn.var <= 255 : "Local variable index to high.";
+        if (varInsn.var <= 6) {
+            writer.writeByte((byte) (OpCode.STLOC0.getValue() + (byte) varInsn.var));
+        } else {
+            writer.writeByte(OpCode.STLOC.getValue());
+            writer.writeByte((byte) varInsn.var);
+        }
+    }
+
+    private static void handleLoadConstant(AbstractInsnNode insn, BinaryWriter writer)
+            throws IOException {
+        assert insn.getType() == AbstractInsnNode.LDC_INSN : "Instruction type doesn't match "
+                + "opcode.";
+
+        LdcInsnNode ldcInsn = (LdcInsnNode) insn;
+        if (ldcInsn.cst instanceof String) {
+            pushDataArray(((String) ldcInsn.cst).getBytes(UTF_8), writer);
+        }
+        // TODO: Handle other types.
+    }
+
+    private static void pushDataArray(byte[] data, BinaryWriter writer) throws IOException {
```

Comment:
How this is different compared to the stuff we have in the `ScriptBuilder` class?